SofaKernel/modules/SofaSimpleFem/src/SofaSimpleFem/HexahedronFEMForceField.h

```diff
@@ -32,6 +32,12 @@
 #include <sofa/helper/decompose.h>
 #include <sofa/helper/OptionsGroup.h>
 
+namespace sofa::component::linearsolver
```

Comment:
It is better to create a fwd.h file for the forward declaration.

Reply:
done. Thanks for the feedback